File: dev-docs/snippets/cron-triggers/delete.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Soft-delete (archive) a cron trigger and cancel its pending alarm.
export async function deleteX(client: JsBaoClient, triggerId: string) {
  // #region example
  const { archived } = await client.cronTriggers.delete(triggerId);
  // #endregion example
  return archived;
}

File: dev-docs/snippets/cron-triggers/delete.swift
import JsBaoClient

// Soft-delete (archive) a cron trigger and cancel its pending alarm.
func delete(client: JsBaoClient, triggerId: String) async throws {
  // #region example
  let result = try await client.cronTriggers.delete(triggerId: triggerId)
  let archived = result.archived
  // #endregion example
  _ = archived
}
